unstructured/partition/lang.py

```diff
@@ -221,7 +223,20 @@ def detect_languages(
     Detects the list of languages present in the text (in the default "auto" mode),
     or formats and passes through the user inputted document languages if provided.
     """
-    if text.strip() == "":
+    if not isinstance(languages, list):
```

Comment:
ah, since the language type check is now here, can we make pdf and image partitioners consistent (either include the typing error or remove it in both places)

Reply:
Actually, pdf and image don't rely on `detect_languages`, so a check is needed, but it's only needed in `partition_pdf_or_image`. See initial conversation and the commit I just added